src/common/grpc/src/select.rs

```diff
@@ -12,107 +12,18 @@
     UInt32Vector, UInt64Vector, UInt8Vector, VectorRef,
 };
-use snafu::{OptionExt, ResultExt};
+use snafu::OptionExt;
 
-use crate::error::{self, ConversionSnafu, Result};
-
-// TODO(LFC): replace with FlightData
-pub async fn to_object_result(output: std::result::Result<Output, impl ErrorExt>) -> ObjectResult {
-    let result = match output {
-        Ok(Output::AffectedRows(rows)) => Ok(ObjectResultBuilder::new()
-            .status_code(StatusCode::Success as u32)
-            .mutate_result(rows as u32, 0)
-            .build()),
-        Ok(Output::Stream(stream)) => collect(stream).await,
-        Ok(Output::RecordBatches(recordbatches)) => build_result(recordbatches),
-        Err(e) => return build_err_result(&e),
-    };
-    match result {
-        Ok(r) => r,
-        Err(e) => build_err_result(&e),
-    }
-}
-
-async fn collect(stream: SendableRecordBatchStream) -> Result<ObjectResult> {
-    let recordbatches = RecordBatches::try_collect(stream)
-        .await
-        .context(error::CreateRecordBatchSnafu)?;
-    let object_result = build_result(recordbatches)?;
-    Ok(object_result)
-}
-
-fn build_result(recordbatches: RecordBatches) -> Result<ObjectResult> {
-    let select_result = try_convert(recordbatches)?;
-    let object_result = ObjectResultBuilder::new()
-        .status_code(StatusCode::Success as u32)
-        .select_result(select_result)
-        .build();
-    Ok(object_result)
-}
-
-#[inline]
-fn get_semantic_type(schema: &SchemaRef, idx: usize) -> i32 {
-    if Some(idx) == schema.timestamp_index() {
-        SemanticType::Timestamp as i32
-    } else {
-        // FIXME(dennis): set primary key's columns semantic type as Tag,
-        // but we can't get the table's schema here right now.
-        SemanticType::Field as i32
-    }
-}
-
-fn try_convert(record_batches: RecordBatches) -> Result<SelectResult> {
-    let schema = record_batches.schema();
-    let record_batches = record_batches.take();
-
-    let row_count: usize = record_batches.iter().map(|r| r.num_rows()).sum();
-
-    let schemas = schema.column_schemas();
-    let mut columns = Vec::with_capacity(schemas.len());
-
-    for (idx, column_schema) in schemas.iter().enumerate() {
-        let column_name = column_schema.name.clone();
-
-        let arrays: Vec<_> = record_batches
-            .iter()
-            .map(|r| r.column(idx).clone())
-            .collect();
-
-        let column = Column {
-            column_name,
-            values: Some(values(&arrays)?),
-            null_mask: null_mask(&arrays, row_count),
-            datatype: ColumnDataTypeWrapper::try_from(column_schema.data_type.clone())
-                .context(error::ColumnDataTypeSnafu)?
-                .datatype() as i32,
-            semantic_type: get_semantic_type(&schema, idx),
-        };
-        columns.push(column);
-    }
-
-    Ok(SelectResult {
-        columns,
-        row_count: row_count as u32,
-    })
-}
+use crate::error::{ConversionSnafu, Result};
```

Comment:
Looks like functions in this file are only used in tests. I'm wondering if we can annotate this mod with `#[cfg(test)]`